### DISCUSSION TITLE
Site slowed down drastically as soon as I added a custom domain

### DISCUSSION THREAD

**Discussion by tunztunztunz** [vercel/vercel#6270] [Help]
I have an extremely simple website with a few images on it. While it was under the vercel.com domain it ran really fast. Routing was instant with zero load times for images. However, I bought a domain from Porkbun and switched over the A records and now the site takes around 3s to load between any route even if I've visited this route before. It doesn't look like it has any issue switch images. It's just the entire page takes 3 seconds. Almost as if something is purposely slowing it down. 

What is causing this?

**Comment by fu-sen** [comment]
Is your domain's name server (set the A record) 🐷Porkbun or ▲Vercel?
You may want to change the name server. 
https://vercel.com/docs/custom-domains#vercel-nameservers.

**Reply by tunztunztunz** [reply]
I set the A record and it was slow. After a night the site is performing as I expect. I'm not really sure what was going on.

**Reply by fu-sen** [reply]
It is a temporary phenomenon that occurs when the name server is switched. Even when switching name servers, old servers will continue to be referenced for hours to days.
